src/_pyinstaller_hooks_contrib/hooks/stdhooks/hook-thinc.py

```diff
@@ -0,0 +1,19 @@
+# ------------------------------------------------------------------
+# Copyright (c) 2021 PyInstaller Development Team.
+#
+# This file is distributed under the terms of the Apache License 2.0
+#
+# The full license is available in LICENSE.APL.txt, distributed with
+# this software.
+#
+# SPDX-License-Identifier: Apache-2.0
+# ------------------------------------------------------------------
+
+"""
+Thinc contains data files and hidden imports. This hook was created to make spacy work correctly.
+"""
+from PyInstaller.utils.hooks import collect_data_files, collect_submodules
+
+datas = collect_data_files("thinc")
+hiddenimports = collect_submodules("thinc")
+hiddenimports += 'blis.cy'
```

Comment:
Surely you mean?
```suggestion
hiddenimports += ['blis.cy']
```

Reply:
I didn't, actually. collect_submodules returns a list, so aren't those semantically the same? Happy to change it if you prefer the other way, though.

Reply:
```python
>>> hiddenimports = []
>>> hiddenimports += 'blis.cy'
>>> hiddenimports
['b', 'l', 'i', 's', '.', 'c', 'y']
```

Reply:
That's right. Sorry, wasn't at 100% yesterday. I get that a string is an iterable which is why this happens but what a bizarre design decision in the language - I'd bet that almost any time somebody writes this they're making the same mistake as me.

Reply:
This line just isn't needed, it must have been an artifact left from when I was figuring out the minimal imports to get spacy to import. I removed it.